### ISSUE TITLE
Can't get clipping from berry bushes

### ISSUE THREAD

**Issue by Gfauf** [slightlycolorblind/Wildcraft#53]
I've tried to to get clippings from bushes by either breaking them with a knife, or using shears but they don't drop anything.
I'm on version 1.18.8 (Stable) of the game, using the following mods:

Acorns v0.5.3 for VS 1.18
ACulinaryArtillery 1.0.14
AnvilMetalRecovery_V0.1.19-pre.1_Debug
CarryOn-1.18-net7_v1.5.0
chiseltools1.8.4
drbraziers_1.1.1
ExpandedFoods 1.6.7
PrimitiveSurvival3.2.7
wildcraft-v1.6.6

I added the wildcraft mod in an already created world, didn't try creating a new one with the mod already installed to test it.

**Comment by RogueRaiden** [comment]
You try while the bush is flowering?

**Comment by Gfauf** [comment]
Yeah, tried while they are flowering and while they're actually harvestable.
Tried breaking with shears, no drop, right clicking with shears, the bush is cropped with no sound effect but no drop, right clicking with knife, nothing, and breaking with knife also nothing.

**Comment by Gfauf** [comment]
Correction: While they're harvestable, if I break them with left click holding shears or knife I get the bush like in vanilla, no clippings though.
